packages/hash/frontend/src/blocks/page/BlockContextMenu/LoadEntityMenuContent.tsx

```diff
@@ -80,49 +71,8 @@ export const LoadEntityMenuContent: FunctionComponent<
     [blockView, blockEntityId],
   );
 
-  const filteredEntities = useMemo(() => {
-    const uniqueEntityIds = new Set();
-    return entities.filter((entity) => {
```

Comment:
@CiaranMn  I want to point out we removed a fair bit of the top-level code for the `LoadEntityMenu` which I assume the data mapping demo was based on

Reply:
It was the entities for the 'swap entity' context menu item (i.e. change which entity is loaded into a block) – I've captured a [task](https://app.asana.com/0/1201095311341924/1203438226339015/f) (internal) to fix it

Reply:
Thank you :)